src/codegen/utilities/function_helpers.py

```diff
@@ -0,0 +1,48 @@
+"""This contains the helper methods used in functions generation."""
+from codegen.functions.function import Function
+from codegen.utilities.helpers import camel_to_snake_case
+
+
+def get_functions(metadata):
+    """Converts the scrapigen metadata into a list of functions."""
+    all_functions = metadata["functions"]
+    functions_metadata = []
+    for function_name, function_data in all_functions.items():
+        if "python_class_name" in all_functions[function_name] and all_functions[function_name]["python_class_name"] == "AIChannelCollection":
```

Comment:
Why is the `AIChannelCollection` hardcoded? can we have the class name as input to the method so that the callers can specify the class name.

Reply:
Updated.